shared/constants/types/config.tsx

```diff
@@ -20,8 +21,6 @@ export type ConnectionType = NetInfo.NetInfoStateType | 'notavailable'
 
 export type State = {
   loggedIn: boolean
-  osNetworkOnline: boolean
-  outOfDate?: OutOfDate
```

Comment:
we make this not undefined as it passes to the menu which will remove items when sending across the bridge so we can't differentiate between that or it being unset